engines/tinsel/handle.cpp

```diff
@@ -21,6 +21,7 @@
  */
 
 #define BODGE
+#define FORBIDDEN_SYMBOL_EXCEPTION_strncmp
```

Comment:
Can we avoid this?

Reply:
As somaen said, we'd better avoid `strncmp()`, and use functions from Common::String. Is it necessary to compare parts of strings? If not, a simple == comparison using Common::String woild suffice

Reply:
I used Common::String now.